Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
```

Comment:
I'll probably change this to alpine or something, but `scratch` makes me so happy